src/Arcus.EventGrid.Tests.Core/Events/Data/CarEventData.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Arcus.EventGrid.Tests.Core.Events.Data
+{
+    public class CarEventData
+    {
+        public string LicensePlate { get; set; }
```

Comment:
Should be made readonly?

Reply:
I fear it might need a setter for (de)serialization. It always comes down to that default constructor 😨

Reply:
And it's just a test class as well 🤷‍♂️

Reply:
Sigh... *thinks about F#*

Reply:
Don't see the problem with this to be honest :)